relay-server/src/actors/project_cache.rs

```diff
@@ -193,6 +155,11 @@ pub struct FetchProjectState {
     pub no_cache: bool,
 }
 
+impl Message for FetchProjectState {
+    type Result = Result<ProjectStateResponse, ()>;
```

Comment:
This requires `ProjectStateResponse`, so that type should be declared before this implementation. That said, you have a TODO to remove this type, so depending on whether you feel this can still go into the changeset feel free to remove this now.

Reply:
Can't remember why I wanted to remove it.

Reply:
It's a pretty redundant type, but there's not a lot of use in either keeping nor removing it.